doc/rtd/topics/integration_tests.rst

```diff
@@ -59,7 +59,7 @@ Cloud Interaction
 Cloud interaction happens via the
 `pycloudlib <https://pycloudlib.readthedocs.io/en/latest/index.html>`_ library.
 In order to run integration tests, pycloudlib must first be
-`configured <https://pycloudlib.readthedocs.io/en/latest/configuration.html#configuration>`_.
+`configured <topics/configuration.rst>`_.
```

Comment:
I just noticed that this wasn't pointing to the current hosted docs - do I need to revert this change?